Html/exercise-04/script.js

```diff
@@ -10,8 +10,10 @@ const user = {
 */
 
 const printUserProfile = ({ name, designation, company, hobbies } = user) => {
+
+    let hobb = hobbies.map((hobby)=>{return hobby})
```

Comment:
this would return the same array right? find an alternate array method that would print _Reading, Listening to music and Collecting stamps_